delivery/services/staging_service.py

```diff
@@ -146,6 +148,9 @@ def stage_order(self, stage_order):
                                  "staging_repo": self.staging_repo,
                                  "session_factory": self.session_factory}
 
+            if not self.file_system_service.exists(stage_order.staging_target):
+                self.file_system_service.makedirs(stage_order.staging_target)
```

Comment:
Not really necessary, is it? Still, won't hurt I guess.

Reply:
It is actually, since it turns out the rsync will only create one levels of directories on the target.

Reply:
aha, in that case: great!